_datafiles/world/default/mobs/frostfang/scripts/26-frostfang_citizen-locketsadness.js

```diff
@@ -48,13 +48,6 @@ function onAsk(mob, room, eventDetails) {
 
     }
 
-    match = UtilFindMatchIn(eventDetails.askText, lichSubjects);
```

Comment:
This seems unrelated to pathfinding?

Reply:
I think something got weird when I rebased on master... that change is already in the master branch.